templates/default/rt/rtsearch.html

```diff
@@ -148,8 +139,7 @@ <H1>{$layout.pagetitle}</H1>
 	</TR>
 	<TR>
 		<TD class="bold nobr">
-			<IMG src="img/report.gif" alt="">
-			{trans("Priority:")}
+			{icon name="priority" label="Priority" fw=true}
```

Comment:
Przypomnij co oznacza `label=""` dla `{icon}`?

Reply:
Parametr label to widzialna etykietka ikonki umiejscowiona obok.
Przykład: https://github.com/chilek/lms-plus/blob/master/templates/default/rt/rtticketinfobox.html#L300